### DISCUSSION TITLE
core-js@<3 is no longer maintained and not recommended for usage due to the number of issues

### DISCUSSION THREAD

**Discussion by theprobugmaker** [vercel/next.js#11058] [Help] [answered]
I just installed Next.js and got this warning. Can I just ignore it? Is it planed to update this in an upcoming version?

Thank you.

**Comment by willemliu** [comment, marked as answer]
Tim posted this answer to an issue about the same concerns. https://github.com/zeit/next.js/issues/8103#issuecomment-586142928

**Reply by timneutkens** [reply]
It's no longer being used (even though it didn't have any issues for what it was used for in Next.js, so the message was significantly too excessive).

This is the last bit to get rid of it completely: https://github.com/zeit/next.js/pull/11070

**Reply by theprobugmaker** [reply]
That is amazing, thank you @timneutkens and @willemliu for the answer.
